gke/nomad/main.tf

```diff
@@ -105,19 +105,6 @@ resource "google_compute_firewall" "nomad_ssh" {
   network       = var.network_name
 }
 
-resource "google_compute_firewall" "nomad_job_ssh" {
```

Comment:
Why was this `nomad_job_ssh` section here? Was there ever a variable called `ssh_allowed_cidr_blocks`?

Reply:
It allowed SSH'ing into jobs running on Nomad clients, but didn't apply to jobs running on VMs, so I chose to use one rule to ~ring them all~ combine them. And yes, there was such a variable - it's the `allowed_cidr_blocks` var from the parent module. Generally speaking, terraform will scream at you if you use a variable that hasn't been declared.